tests/appsec/iast/taint_sinks/test_sql_injection.py

```diff
@@ -26,6 +35,9 @@
 ]
 
 
+patch(pymysql=True, mysqldb=True)
```

Comment:
mmmm, I don't like that global, could you move this patch to a `pytest_configure()` or a `@pytest.fixture(autouse=True)`?

Reply:
This actually shouldn't be required, so I don't want to hide it